File: 250212/문자열 길이의 합/sum-length-of-string.js
const fs = require("fs");
const input = fs.readFileSync(0).toString().trim().split("\n");
let inputIndex = 0;

const n = parseInt(input[inputIndex++]);
let totalLen = 0;
let startsWithA = 0;

for (let i = 0; i < n; i++) {
    let str = input[inputIndex++];
    if (str[0] == 'a')
        startsWithA++;
    totalLen = totalLen + str.length;
}

console.log(totalLen, startsWithA);

File: 250212/문자열 길이의 합/sum-length-of-string.java
import java.util.*;
import java.io.*;

public class Main {
    public static void main(String[] args) throws Exception {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));
        StringBuilder sb = new StringBuilder();
        StringTokenizer st;
        int totalLen = 0, countStartsWithA = 0;

        st = new StringTokenizer(br.readLine());
        int n = Integer.parseInt(st.nextToken());
        for (int i = 0; i < n; i++) {
            st = new StringTokenizer(br.readLine());
            String str = st.nextToken();
            totalLen = totalLen + str.length();
            if (str.charAt(0) == 'a') countStartsWithA++;
        }
        sb.append(String.valueOf(totalLen)).append(" ").append(String.valueOf(countStartsWithA));
        bw.append(sb.toString());
        bw.close();
    }
}